views/student-edit-page.hbs

```diff
@@ -0,0 +1,228 @@
+<div id="wrapper">
+  <div id="sidebar-wrapper">
+    <ul class="sidebar-nav">
+      <li class="sidebar-brand">
+        <img class="sidebar-logo" src="/images/Logo-horizontal-white.png" />
+      </li>
+      <li>
+        <a href="/dashboard">Dashboard</a>
+      </li>
+      <li>
+        <a href="/class">Class</a>
+      </li>
+      <li>
+        <a href="/teacher">Teacher</a>
+      </li>
+      <li>
+        <a href="/subject">Subject</a>
+      </li>
+      <li>
+        <a class="student-page" href="/student">Student</a>
+      </li>
+      <li>
+        <a href="/approvals">Approvals</a>
+      </li>
+    </ul>
+  </div>
+  <!-- /#sidebar-wrapper -->
+
+  <!-- Page Content -->
+  <div id="page-content-wrapper">
+
+    <div class="container-nav">
+      <ul
+        class="nav justify-content-end bg-light toggle-float-left custom-nav-setting"
+      >
+        <abutton
```

Comment:
`<abutton` is not an html property. Do you mean `<a>` as you have the `href` attribute?